snap/snapcraft.yaml

```diff
@@ -44,6 +44,7 @@ apps:
       - network
       - network-bind
       - network-control
+      - network-setup-control
```

Comment:
With the additional plug, [this doc section](https://github.com/canonical/microovn/blob/d49dc8f1574ec927fcdcf15e783507641d0cdec3/docs/developers/building.rst?plain=1#L87-L96) needs an update as well.